pkg/yurtctl/util/edgenode/exec.go

```diff
@@ -0,0 +1,76 @@
+package edgenode
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os/exec"
+	"strings"
+	"syscall"
+)
+
+type Command struct {
+	Cmd      *exec.Cmd
+	StdOut   []byte
+	StdErr   []byte
+	ExitCode int
+}
+
+func NewCommand(command string) *Command {
+	return &Command{
+		Cmd: exec.Command("bash", "-c", command),
+	}
+}
+
+// Exec run command and exit formatted error, callers can print err directly
+// Any running error or non-zero exitcode is consider as error
+func (cmd *Command) Exec() error {
```

Comment:
why not use `os/exec` package?

Reply:
The struct `Command`  contains all the information about cmd running and can be used in other places. If wants to be simpler, os/exec can be uses here.